pkg/operator/podsecurityreadinesscontroller/podsecurityreadinesscontroller_test.go

```diff
@@ -189,3 +261,49 @@ func TestPodSecurityViolationController(t *testing.T) {
 		})
 	}
 }
+
+func TestNamespaceSelector(t *testing.T) {
```

Comment:
we are indirectly testing namespace selector logic here, but makes me feel safer.